compose: new page for ingestion actions. juggling ingestion content a… (#7903)
umbraco/UmbracoDocs: umbraco-compose/apis/ingestion/restful-ingestion.md

# RESTful Ingestion

> In many scenarios you may wish to create an integration with Umbraco Compose from an application which has control over the payload sent to the Ingestion API.

Reviewer: ```suggestion
You can integrate Umbraco Compose with applications that control the payload sent to the Ingestion API.
```
